UNRELEASED.md

```diff
@@ -1 +1,7 @@
 # Unreleased
+
+**Breaking:**
+
+- react-native-bpk-component-navigation-bar:
+  - New `type` prop for `BpkNavigationBarTextButtonIOS` to change the button colour.
+  - Added `navigationBarPrimaryColor` theme property, applied to `BpkNavigationBarTextButtonIOS` when type is `primary`. This is a breaking change as you must now supply this theme property to theme the component.
```

Comment:
I think this could be clearer to make consumers know that it is only breaking iff they are applying a theme

Reply:
Thanks - how about now? I added this to the end:

> If you are not using theming, this release is not breaking.